src/components/extensive/Tables/NurseriesTable.tsx

```diff
@@ -57,8 +50,13 @@ const NurseriesTable = ({ project, onFetch, hasAddButton = true }: NurseriesTabl
         primaryButtonProps={{
           children: t("Yes"),
           onClick: () => {
-            deleteNursery({ pathParams: { uuid } });
-            closeModal(ModalId.CONFIRM_NURSERY_DELETION);
+            deleteNursery(uuid)
+              .then(() => {
+                nurseryIndex.refetch();
```

Comment:
We get some magic here - this refetch is no longer needed; When the delete is recorded in the v3 store, the index connection will note that one of its records is missing and refetch automatically. I would streamline this method like so:
```
onClick: async () => {
  await deleteNursery(uuid);
  closeModal(ModalId.CONFIRM_NURSERY_DELETION);
}
```